src/common/main.lua

```diff
@@ -1,28 +1,143 @@
 local json = require(".common.json")
 local utils = require(".common.utils")
 local main = {}
+-- just to ignore lint warnings
+local ao = ao or {}
 
 local camel = utils.camelCase
+
+local function errorHandler(err)
+	return debug.traceback(err)
+end
+
 main.init = function()
-	Owner = Owner or ao.env.Process.Owner
-	Name = Name or "ar.io ANT Registry"
+	ANTS = ANTS or {}
+	-- Example ANT structure
+	-- ANTS["antId"] = {
+	--     Owner = "userId",
+	--     Controllers = {"userId1", "userId2"},
+	-- }
+	USERS = USERS or {}
+	-- Example USERS structure
+	-- USERS["userId"] = {"antId1", "antId2"}
```

Comment:
`"antId1", "antId2"` makes me think there's going to be strings there instead of ANT object references... (reading rest of PR now to see if we somehow ended up there instead).